### ISSUE TITLE
Add property for latest stable React release to be used with React settings

### ISSUE THREAD

**Issue by bennyn** [yannickcr/eslint-plugin-react#1981]
There is this warning when not specifying a React version in `.eslint.js`:

> Warning: React version not specified in eslint-plugin-react settings. See https://github.com/yannickcr/eslint-plugin-react#configuration.

Telling from the [documentation](https://github.com/yannickcr/eslint-plugin-react#configuration), I can see that the default is to use the latest stable React release if no version is specified. That's great because in my case I always run on the latest stable release, so the default option is what suits me best. However, there is a warning when running `eslint` because I have no version specified in my config.

From the [implementation](https://github.com/yannickcr/eslint-plugin-react/blob/v7.11.1/lib/util/version.js#L12) I can see that the default version is just set to `'999.999.999'`, so I can fix the warning (and get the default behaviour) by adding this to my `.eslint.js`:

```json
"settings": {
  "react": {
    "version": "999.999.999"
  }
}
```

It would be much nicer though to declare `"version": "latest"` instead of `"version": "999.999.999"`.

**Comment by ljharb** [comment]
Duplicate of #1955.
